backend/app/services/workflow/service.py

```
@@ -622,6 +636,68 @@ async def get_community_workflows(
+                    "creator": {
+                        "id": workflow.get("created_by"),
+                        "name": creator_info.get("name", "GAIA Team"),
+                        "avatar": creator_info.get("picture"),
+                    },
+                }
+                formatted_workflows.append(formatted_workflow)
+
+            return PublicWorkflowsResponse(workflows=formatted_workflows, total=total)
+
+        except Exception as e:
+            logger.error(f"Error fetching explore workflows: {str(e)}")
```

Ruff (0.14.5)

Use explicit conversion flag

Replace with conversion flag

(RUF010)